### PR TITLE
reduced margin top down to 5px from 24px

### LINKED ISSUE #4610: Unnecessary padding 24px above config options in product card on PLP
**Description**:
Unnecessary padding 24px above config options in product card on PLP

**Expected behavior**:
Remove padding 24px
Screenshot 2022-03-29 at 12 59 52

**To Reproduce**:
1. Open PLP
2. Hover mouse on config product card

**Actual result**:
Unnecessary padding 24px above config options 
Screenshot 2022-03-29 at 12 34 47

### PR DESCRIPTION
**Pull request by AzizKHAN030**
**Related issue(s):**
* Fixes #4610 

**Problem:**
* Unnecessary margin-top for ProductConfigurableAttributes block

**In this PR:**
* Reduced the margin down to 5px because on hover to the attribute, the border should be visible

### CODE REVIEW

**Comment by AzizKHAN030** [comment]
Didn't get rid of the margin completely because on hover to attribute the appeared border was partially hidden under top block
image

File: packages/scandipwa/i18n/da_DK.json [hunk 1]
```diff
@@ -589,11 +589,13 @@
...
     "Unable to reset password": null,
     "Please enter no more than %s characters.": null,
     "You saved the account information.": null,
     "Minimum of different classes of characters in password is %s. ": null,
-    "Classes of characters: Lower Case, Upper Case, Digits, Special Characters.": null
-}
+    "Classes of characters: Lower Case, Upper Case, Digits, Special Characters.": null,
+    "Maximum %s characters (%s remaining)": null,
+    "Maximum %s characters": null,
+    "Maximum %s characters (%s too many)": null
```
Comment (zans-laksa) [inline packages/scandipwa/i18n/da_DK.json:600]: Usually, when localization files are updated, they all are updated, not just the Danish one.

**Review by zans-laksa — CHANGES_REQUESTED** [review]

**Comment by zans-laksa** [comment]
If you fix something in another PR, please provide a link to this PR.

**Comment by AzizKHAN030** [comment]
This issue was fixed by another person, not in my PR

**Pull request closed without merging**